src/routes/clientRoutes.js

```diff
@@ -23,5 +23,6 @@ router.delete(
   clientController.deleteFavoriteDish
 );
 router.get('/chefs', isUserCreatedClient, clientController.getChefs);
+router.post('/:id', isUserCreatedClient, clientController.dishesRatings);
```

Comment:
We need to change the path name, cuz it's confusing now. I think what makes more sense in this case is this:
`router.post('/rate-dish/:id', isUserCreatedClient, clientController.dishesRatings);`